tests/end2end/features/tabs.feature

```diff
@@ -255,6 +255,16 @@ Feature: Tab management
             - data/numbers/2.txt (active)
             - data/numbers/3.txt
 
+    Scenario: :tab-focus with 0
+        When I open data/numbers/1.txt
+        And I open data/numbers/2.txt in a new tab
+        And I open data/numbers/3.txt in a new tab
+        And I run :tab-focus 0
```

Comment:
This still checks the argument and not the count, it should probably be "And I run :tab-focus with count 0" instead.

Also, the last tab is already focused by default, so I guess you'd need to do "And I run :tab-focus 1" or so first for the test to be effective.